f/radio/fn_radioChannels.sqf

```diff
@@ -85,5 +85,35 @@ if (isServer) then {
 // Run clientside stuff
 if (hasInterface) then {
 	[] call f_fnc_radioAddHandlers;
+	
+	private _personalRadioCode = "
+		private _radioOn = player getVariable [""f_var_radioIsOn"",true];
+		player setVariable [""f_var_radioIsOn"",!_radioOn];
+		[player] spawn f_fnc_radioCheckChannels;
+	";
+	
+	private _vehicleRadioCode = "
+		if (!isNull objectParent player) then {
+			private _radioOn = (objectParent player) getVariable [""f_var_radioIsOn"",true];
+			(objectParent player) setVariable [""f_var_radioIsOn"",!_radioOn];
+			[player] spawn f_fnc_radioCheckChannels;
+		};
+	";
+	
+	waitUntil {scriptDone f_script_briefing};
+	
+	player createDiaryRecord ["fa3_actions","FA3 Radio",
```

Comment:
should be player createDiaryRecord ["fa3_actions",["FA3 Radio","all the briefing jazz"]; I think